website/docs/Installation.md

```diff
@@ -1,10 +1,57 @@
 
-### Option 1: venv
+#### Step 1. Install Docker.
+
+Install docker following [this instruction](https://docs.docker.com/get-docker/).
+
+For Mac users, alternatively you may choose to install [colima](https://smallsharpsoftwaretools.com/tutorials/use-colima-to-run-docker-containers-on-macos/) to run docker containers, if there is any issues with starting the docker daemon.
+
+#### Step 2. Build a docker image
+
+AutoGen provides [dockerfiles](https://github.com/microsoft/autogen/tree/main/samples/dockers/) that could be used to build docker images. Use the following command line to build a docker image named `autogen_img` (or other names you prefer) from one of the provided dockerfiles named `Dockerfile.base`:
+
+```
+docker build -f samples/dockers/Dockerfile.base -t autogen_img https://github.com/microsoft/autogen.git#main
+```
+which includes some common python libraries and essential dependencies of AutoGen, or build from `Dockerfile.full` which include additional dependencies for more advanced features of AutoGen with the following command line:
+
+```
+docker build -f samples/dockers/Dockerfile.full -t autogen_full_img https://github.com/microsoft/autogen.git
+```
+Once you build the docker image, you can use `docker images` to check whether it has been created successfully.
+
+#### Step 3.  Run applications built with AutoGen from a docker image.
+
+**Save keys into environment variable:**
+If your application needs an oai key, save your oai key into an environment variable `OPENAI_API_KEY`, which can be done by adding the following line in your `~/.bashrc` (or `~/.zshrc`),
+
+```
+export OPENAI_API_KEY="sk-xxxxxx"
+```
+
+**Mount your code to the docker image and run your application from there:** Now suppose you have your application built with AutoGen in a folder named `myapp`. You can mount it into the docker image and run it. In the example below, the folder `myapp` is mounted into `/myapp` in the docker, and the script `main_twoagent.py` is executed in the docker.
+
+```python
+# Mount the local folder `myapp` into docker image and run the script in the docker.
+docker run -it -e OPENAI_API_KEY=$OPENAI_API_KEY -v `pwd`/myapp:/myapp autogen_img:latest python /myapp/main_twoagent.py
+```
+
+Similarly, you may also run [AutoGen Studio](https://github.com/microsoft/autogen/tree/main/samples/apps/autogen-studio) (assuming that you have built a docker image named `autogen_full_img` with `Dockerfile.full`) as below:
+
+```
+docker run -it -e OPENAI_API_KEY=$OPENAI_API_KEY -p 8081:8081 autogen_full_img:latest autogenra ui --host 0.0.0.0
```

Comment:
avoid exporting env var.
@victordibia is it avoidable for autogen studio?

Reply:
Some tests fail if OPENAI_API_KEY is not defined. We should probably make those tests use OAI_CONFIG_LIST and then there is no need to have OPENAI_API_KEY defined.